portfolio/src/main/webapp/index.html

```diff
@@ -19,8 +19,8 @@
 			<div class="hero-text">
 				<h1>I am Zhamilya (Jamila) Sabazova</h1>
```

Comment:
indenting is off in this file. I think you might need to change some setting on your editor, it seems to want to put tabs in instead.

Reply:
fixed

Reply:
there's still tabs all through this file, please turn tabs off and remove them

Reply:
I think now it should be fixed